bokehjs/src/coffee/models/tools/gestures/wheel_pan_tool.coffee

```diff
@@ -6,18 +6,7 @@ import * as p from "../../../core/properties"
 export class WheelPanToolView extends GestureToolView
 
   _scroll: (e) ->
-    # we need a browser-specific multiplier to have similar experiences
-    if navigator.userAgent.toLowerCase().indexOf("firefox") > -1
-      multiplier = 20
-    else
-      multiplier = 1
-
-    if e.originalEvent?.deltaY?
-      delta = -e.originalEvent.deltaY * multiplier
-    else
-      delta = e.bokeh.delta
```

Comment:
@bryevdv, it works on Linux, at least in Chrome 51 and FF 41 & 47. My biggest concern was Safari, but if that works we should be good. Still I would like to test a range of browser versions, but I don't have a setup for this right now. On a side note, how hard would it be to have some browser versions as conda packages, so I would easily switch between them as currently I do with Python versions?